resources/assets/js/src/components/pages/Login.tsx

```diff
@@ -1,24 +1,32 @@
 import * as React from "react";
-import {Button, Col, ControlLabel, form, FormControl, FormGroup, Row} from "react-bootstrap";
+import {Button, Col, ControlLabel, Form, FormControl, FormGroup, HelpBlock, Row} from "react-bootstrap";
 import {connect} from "react-redux";
 import { Redirect } from "react-router-dom";
```

Comment:
extra spaces

Reply:
What do you mean?

Reply:
{connect} vs  { Redirect } , use the same format for everyone e.g. {Redirect}